src/test/java/api/support/fixtures/policies/PoliciesActivationFixture.java

```diff
@@ -0,0 +1,226 @@
+      .noticePolicy(noticePoliciesFixture.inactiveNotice()));
+  }
+
+  /**
+   * This method uses the loan policy, allowAllRequestPolicy request policy,
+   * activeNotice notice policy, facultyStandard overdue fine policy from
+   * the loanPolicyBuilder.
+   * @param loanPolicyBuilder - loan policy builder.
+   */
+  public void use(LoanPolicyBuilder loanPolicyBuilder) {
+    use(rollingDefaultPolicies()
+      .loanPolicy(loanPoliciesFixture.create(loanPolicyBuilder)));
+  }
+
+  public void use(PoliciesActivation.PoliciesActivationBuilder builder) {
+    final PoliciesActivation policies = builder.build();
+
+    useFallbackPolicies(policies.getLoanPolicy().getId(),
+      policies.getRequestPolicy().getId(),
+      policies.getNoticePolicy().getId(),
+      policies.getOverduePolicy().getId(),
+      policies.getLostItemPolicy().getId());
+  }
+
+  /**
+   * This method uses the loan policy, allowAllRequestPolicy request policy,
+   * activeNotice notice policy, facultyStandard overdue fine policy from
+   * the loanPolicyBuilder and noticePolicyBuilder.
+   * @param loanPolicyBuilder - loan policy builder.
+   */
+  public void use(LoanPolicyBuilder loanPolicyBuilder,
+    NoticePolicyBuilder noticePolicyBuilder) {
+
+    use(rollingDefaultPolicies()
+      .loanPolicy(loanPoliciesFixture.create(loanPolicyBuilder))
+      .noticePolicy(noticePoliciesFixture.create(noticePolicyBuilder)));
+  }
+
+  /**
+   * This method uses notice policy, canCirculateRolling loan policy,
+   * allowAllRequestPolicy request policy,
+   * facultyStandard overdue fine policy from
+   * the loanPolicyBuilder.
+   *
+   * @param noticePolicy - notice policy.
+   */
+  public void use(NoticePolicyBuilder noticePolicy) {
+    use(rollingDefaultPolicies()
+      .noticePolicy(noticePoliciesFixture.create(noticePolicy)));
+  }
+
+  public void useLostItemPolicy(UUID lostItemFeePolicyId) {
+    final PoliciesActivation policies = rollingDefaultPolicies().build();
+
+    useFallbackPolicies(policies.getLoanPolicy().getId(),
+      policies.getRequestPolicy().getId(),
+      policies.getNoticePolicy().getId(),
+      policies.getOverduePolicy().getId(),
+      lostItemFeePolicyId);
+  }
+
+  /**
+   * This method uses the loan policy, allowAllRequestPolicy request policy,
+   * activeNotice notice policy, facultyStandard overdue fine policy from
+   * the loanPolicyBuilder.
+   * @param loanPolicyBuilder - loan policy builder.
+   */
+  public void useWithActiveNotice(LoanPolicyBuilder loanPolicyBuilder) {
+    use(rollingDefaultPolicies()
+      .loanPolicy(loanPoliciesFixture.create(loanPolicyBuilder)));
+  }
+
+  /**
+   * This method uses notice policy, canCirculateRolling loan policy,
+   * allowAllRequestPolicy request policy, facultyStandard overdue fine policy from
+   * the loanPolicyBuilder.
+   * @param noticePolicy - notice policy.
+   */
+  public void useWithPaging(NoticePolicyBuilder noticePolicy) {
```

Comment:
Does this only allow page requests to be created, or does the default request policy disallow them and this allows page and other types of requests?

The comments seem to be inconsistent with the implementation, they refer to the allow all request policy, whereas the code refers to the page request policy.

```suggestion
  public void useDefaultRollingPoliciesAndOnlyPageRequests(NoticePolicyBuilder noticePolicy) {
```

Reply:
The default policy allows all request policies but this method allows only PAGE requests.

Reply:
I have renamed the method.